.nycrc

```diff
@@ -0,0 +1,11 @@
+{
+  "include": [
+    "**/**/src/*.ts"
+  ],
+  "exclude": [
+    "**/**/**/*.spec.js"
+  ],
+  "lines": 95,
```

Comment:
why 95 and not 100?

Reply:
For a large enough file, that can be multiple if statements / loops etc.

Reply:
I can bump it up however I think 100% is not necessary. Google's general guidance is that 90% coverage is "exemplary". 
When we log an error with process.env. that line is not going to be tested.

Reply:
Hmm ok.